src/main/kotlin/com/terraformation/backend/accelerator/db/ActivityStore.kt

```diff
@@ -97,47 +97,64 @@ class ActivityStore(
 
   fun update(
       activityId: ActivityId,
+      isAdmin: Boolean = false,
```

Comment:
There's enough conditional logic hanging off this flag that I suspect it'd be better to split this into two functions.